proteus/FemTools.py

```diff
@@ -7740,3 +7740,228 @@ def __init__(self,
 #  #end checkFunctionTrace
     #check if contiuous solution has no jumps?
     #checkFunctionTrace(u,mesh,globalDGspace,traceOp)
+
```

Comment:
OK, to clarify: my previous comment on these  was  to move them into MeshTools, not FemTools. There is a comment bit of abused  terminology in the FEM world that we continue to abuse. I "Finite Element" is the mathematical abstraction defined by the Ciarlet triple. We also use "Element" to to mean the mesh cells and ReferenceElement to mean the reference cell used  in the parametric finite element approach, where things are defined on a reference geometry and mapped  to the physical cells. So the stuff you're doing here is really operating on mesh cells and geometry, so I was thinking it's more natural to keep it  in MeshTools.py